modules/vpc/main.tf

```diff
@@ -30,10 +30,7 @@ resource "aws_subnet" "Private" {
     Name = "private${count.index}"
   }
 }
-output "private_subnet_id" {
-  description = "Private Subnet ID"
-  value       = aws_subnet.Private.*.id
-}
```

Comment:
Please don't mix changes not related to the feature in a single PR

Reply:
resolved!

Reply:
Are you sure, I still see this change from my side